### PR TITLE
Use released version of Flipflop

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

**Comment by mjgiarlo** [comment]
@jcoyne Done. Let's see what Travis does with this.

File: app/controllers/sufia/admin/features_controller.rb [hunk 1]
```diff
@@ -2,11 +2,18 @@ module Sufia
...
       before_action do
         authorize! :manage, Sufia::Feature
+      end
+
+      # overriding so we can have a layout https://github.com/voormedia/flipflop/issues/18
+      def index
         add_breadcrumb t(:'sufia.controls.home'), root_path
         add_breadcrumb t(:'sufia.toolbar.admin.menu'), sufia.admin_path
         add_breadcrumb t(:'sufia.admin.sidebar.settings'), sufia.admin_features_path
+        @feature_set = Flipflop::FeaturesController::FeaturesPresenter.new(Flipflop::FeatureSet.current)
```
Comment (jcoyne) [inline app/controllers/sufia/admin/features_controller.rb]: This should not be necessary. Just call super.

**Comment by mjgiarlo** [comment]
@jcoyne Good catch. I think I've got it now if you care to re-review.

File: app/controllers/sufia/admin/features_controller.rb [hunk 2]
```diff
@@ -2,11 +2,17 @@ module Sufia
   module Admin
     class FeaturesController < Flipflop::FeaturesController
       layout 'admin'
+
       before_action do
         authorize! :manage, Sufia::Feature
+      end
+
+      # overriding so we can have a layout https://github.com/voormedia/flipflop/issues/18
```
Comment (jcoyne) [inline app/controllers/sufia/admin/features_controller.rb]: Can we remove this comment?

**Pull request merged**